content/zh/docs3-building/docs/references/metadata/nacos.md

```diff
@@ -0,0 +1,618 @@
+---
```

Comment:
merge this file into `content/zh/docs3-building/java-sdk/reference-manual/metadata-center/nacos/guide.md`